starter-repos/app-template/config/envs/prod-central/deployment.yaml

```diff
@@ -7,8 +7,16 @@ spec:
     spec:
       containers:
       - name: app
+        # Please change the image to your own container registry address.
+        # "image" has to be placed in each individual deployment.yaml 
+        # rather than in the ../base/app/deployment.yaml. This allows 
+        # the skaffold kubectl deployer to attach the image digest
+        image: app
         env:
         - name: ENVIRONMENT
           value: prod-central
-
-
+        - name: REDISHOST
+          valueFrom:
+            configMapKeyRef:
+              name: redishost
+              key: REDISHOST
```

Comment:
Can't `REDISHOST` be in the `base`?

Reply:
no. If REDISHOST is put in the base, it would requires the kustomization.yaml rule for the configMapKeyRef (which requires the metadata.namespace field, which blocks skaffold to use the namespace from kubens).

Reply:
We can add it as a patch in `config/base/kustomization.yaml`. Then it doesn't affect `config/base/app` or `config/base/redis`.

Reply:
Yes, this is what this CL does. 😄 the kustomization rule for REDISHOST is moved to the `config/base/kustomization.yaml` and the dev refers its resource specifically to `config/base/app` and  `config/base/redis`. Do you suggest the deployment resource (including REDISHOST) can also be defined in the `config/base/kustomization.yaml`

Reply:
Used the patch to host the REDISHOST. Please take a look at the new commit @janetkuo @Liujingfang1